frontend/src/pages/devices/List.tsx

```diff
@@ -123,14 +124,15 @@ export const List = () => {
         warnings.push.apply(warnings, getLockResponsivenessWarnings(entity))
 
         if (warnings.length === 1) {
-            return <>{ warnings[0] }</>
+
+            return <ListGroup.Item className="border-0">{ warnings[0] }</ListGroup.Item>
         }
         return (
-            <ul>
+            <ListGroup className="px-0">
```

Comment:
Would [flush](https://getbootstrap.com/docs/4.0/components/list-group/#flush) work in place of `px-0` and `border-0`?

Reply:
I was able to change 2 of them. The 3rd one gave it a weird border
![image](https://user-images.githubusercontent.com/54916525/196294468-f91fc61c-3b35-4972-8105-baa8c4d4ad56.png)